src/pixee-platform.ts

```diff
@@ -32,7 +33,7 @@ export async function triggerPrAnalysis(prNumber: number) {
         Authorization: `Bearer ${token}`,
       },
     })
-    .then((response) => {
+    .then((_) => {
```

Comment:
I don't think it's necessary to use _

```suggestion
    .then(() => {
```

Reply:
TIL

thanks, fixed